app/services/users.js

```diff
@@ -28,3 +29,20 @@ exports.createSession = async params => {
   logger.error('Password and mail mismatch for user:', params.mail);
   throw errors.authenticationError("The password and mail combination doesn't match");
 };
+
+exports.index = async req => {
+  logger.info('Index Users with: ', req);
+  const page = req.headers.page || 0;
+  const type =
+    (await jwt.findUserByToken(req.headers.authorization)).type === 'regular'
+      ? 'regular'
+      : ['admin', 'regular'];
+  const { count, rows } = await User.findAndCountAll({
+    where: {
+      type
+    },
+    offset: page,
+    limit: config.common.api.paginationLimit
```

Comment:
Why I can't set my limit (it's ok have a upper limit) but maybe I want less records